### ISSUE TITLE
IE10 running code before all libraries are ready

### ISSUE THREAD

**Issue by cormip** [dolox/fallback#19]
The problem is specifically with the fallback.ready() function when the [libraries] array is NOT passed. 

```
fallback.ready(function() {
    // All of my libraries have finished loading!
    // IE10 will execute this code before all libraries have been loaded
    // Execute my code that applies to all of my libraries here!
});
```

Works as expected in IE11, FF & Chrome. Problem was tested in both a real IE10 install as well as using IE11 Debugger <F12> with IE10 emulation enabled.

**Comment by sgarbesi** [comment]
@cormip I'll take a look this weekend. If you have a fix, please make a PR.

**Comment by sgarbesi** [comment]
@cormip Can you please provide a test case on http://plnkr.co/ ?

**Comment by cormip** [comment]
See: http://plnkr.co/edit/oklrBQ0lDqNNcqdJXBNr

IE10 will throw an error: '$' is undefined

Issue appears related to loading of CSS file. If I remove that line, it works without error.

**Comment by sgarbesi** [comment]
@cormip It's IE10 caching related with the callbacks. For the CSS it's making the callback on both `onReadyStateChange` and `onLoad`. If it's a fresh hit, the problem doesn't occur.

Keep you posted.

**Comment by sgarbesi** [comment]
@cormip I patched this bug in v1.1.3.

Please give it a try and confirm before I close this issue.

Tested via BrowserStack and it seems to have fixed the issue.
Thanks for finding/bringing to my attention.

**Comment by cormip** [comment]
Testing via IE11 set to IE10 Document Emulation Mode indicates that the issue has been fixed in v1.1.3.
